src/app/technical-record/weights/weights.component.html

```diff
@@ -3,10 +3,13 @@
   <tr class="govuk-table__row">
     <th scope="col" class="govuk-table__header"></th>
     <th scope="col" class="govuk-table__header">GB weight (kg)</th>
-    <th scope="col" class="govuk-table__header">EEC weight (kg)</th>
+    <th scope="col" class="govuk-table__header" *ngIf="activeRecord.vehicleType === 'hgv' || activeRecord.vehicleType === 'trl'">EEC weight (kg)</th>
     <th scope="col" class="govuk-table__header">Design weight (kg)</th>
-    <th class="govuk-table__cell"></th>
-    <th class="govuk-table__cell"></th>
+    <td scope="col" class="govuk-table__header" *ngIf="activeRecord.vehicleType === 'psv'">Kerb weight (kg)</td>
```

Comment:
We can also probably do the same exercise (body and brakes) here again if we have table format,
vehicle-weight, gross-vehicle-weight, design, axle weight, 
GB, design (might be dependent on axle number field so we have to check the behaviour of what is happening when we add axles, whether it populates a section with GB, Design (and kerb laden for PSV).
We could could be showing the common table head and cells elements
then show the ones for psv
axle number field might be its own separate module as well as it populate the form the other axle fields i believe.